.github/workflows/releasability.yaml

```diff
@@ -0,0 +1,143 @@
+# Copyright 2023 The Knative Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'releasability'
+
+on:
+  workflow_call:
+    inputs:
+      repo:
```

Comment:
The repos here should just be read in from the `repos.yaml` file and matrix across this workflow.